app/main.py

```diff
@@ -1,3 +1,46 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> float:
```

Comment:
```suggestion
    def __add__(self, other: int | float) -> float:
```
you return `Distance`

fix it in other cases

Reply:
also, you take `Distance` as well

Reply:
Do you mean this way: def __add__(self, other: int | Distance) -> Distance: ? But " NameError: name 'Distance' is not defined'', why it occurs and how to fix it?